code/game/objects/items/stacks/stack.dm

```diff
@@ -246,6 +246,13 @@
 				continue
 			if(istype(AM,/obj/structure/table))
 				continue
+			if(istype(AM,/obj/structure/window))
+				var/obj/structure/window/W = AM
+				var/obj/structure/window/O = new R.result_type
+				if(istype(O,/obj/structure/window) && O.fulltile)
+					//Do nuthin and move on
```

Comment:
really...

Reply:
looking at this I realize that I should have compressed them into one if. Doing that now